source/tutorials/configure-selinux/configure-selinux.rst

```diff
@@ -0,0 +1,139 @@
+.. _configure-selinux:
+
+SELinux Configuration and Rules
+###############################
+
+Security-Enhanced Linux (`SELinux <https://source.android.com/security/selinux>`_) is enabled in Android to enforce the Mandatory Access Control for security. SELinux supports two working modes: permissive and enforcing:
+
+* In **permissive** mode, it only audits the operations of all domains and prints the AVC (Access Vector Cache) errors that violate the sepolicy rules, but it never blocks any operations.
+
+* In **enforcing** mode, it not only prints out the AVC errors, but also blocks the operations that violate the sepolicy rules.
+
+It's a good practice to set SELinux in permissive mode at the beginning of the development phase, so that you can easily bring up the system and fix any SELinux errors. After resolving all AVC errors, you can change the SELinux mode to enforcing mode.
+
+SELinux Configuration
+=====================
+
+To change SELinux in build time
+-------------------------------
+
+    The SELinux operation mode is configured using *mixins*. You can edit the '*mixins.spec*' file in the source folder for your lunch target at '*device/intel/project-celadon/<target-name>/mixins.spec*', and rebuilt the image to take effect. Set the '*sepolicy*' key to either **permissive** or **enforcing** to enable the corresponding SELinux mode.
+
+    .. code-block:: none
+        :emphasize-lines: 3
+
+        ...
+        project-celadon: default
+        sepolicy: permissive
+        graphics: project-celadon(gen9+=true,hwc2=true,vulkan=false,drmhwc=false,minigbm
+        ...
+
+    .. note::
+        1. The *permissive* mode only can be activated in **userdebug** or **eng** builds, as the user builds always operate in *enforcing* SELinux mode in Android.
+        2. After you change the *sepolicy* value in *mixins.spec*, you need to run the '*device/intel/mixins/mixin-update*' script to make the changes taking effect before starting the build.
```

Comment:
Suggested rewrite: 
2. After you change the *sepolicy* value in *mixins.spec*, run the '*device/intel/mixins/mixin-update*' script to make the changes take effect before the build starts.